src/main/resources/templates/mdBenefitsFlow/householdIncomeWho.html

```diff
@@ -24,15 +24,15 @@
                             inputName='householdMemberJobAdd',
                             content=~{::householdMembers})}">
                 <th:block th:ref="householdMembers">
+                  <th:block
+                      th:replace="~{fragments/inputs/radio :: radio(inputName=${inputName},value='you', label=${inputData.firstName + ' ' + inputData.lastName + ' ' +  #messages.msg('general.you')})}"/>
```

Comment:
Everything looks good to go but I was curious why this value is you where the other values are the persons name?

Reply:
This is for the person filling out the application.

Reply:
It used to be on line 34, but I moved it to above the other house hold members to match the design.

Reply:
Oh, I left this part alone so I didn't have to update the pdf logic, as that looks for "you" explicitly.